ocaml/xapi/xapi_pool_helpers.ml

```diff
@@ -61,7 +62,8 @@ let valid_operations ~__context record _ref' =
     | [_] -> set_errors Api_errors.cluster_already_exists [] [ `cluster_create ]
     (* indicates a bug or a need to update this code (if we ever support multiple clusters in the pool *)
     | _::_ -> failwith "Multiple clusters exist in the pool"
-    | [] -> if ha_enabled then raise (Api_errors.Server_error(Api_errors.incompatible_cluster_stack_active, ["xhad"]))
+    (* cluster create cannot run if ha is already enabled *)
+    | [] -> if ha_enabled then raise (Api_errors.Server_error(Api_errors.incompatible_cluster_stack_active, [current_stack]))
```

Comment:
Sorry, one more change, I think you need to use `set_errors .... [`cluster_create]` here, otherwise you'll always fail if HA is enabled and there is no clustering